### PR TITLE
Added questing + crafting stats to the legions collection pages

### PR DESCRIPTION
**Pull request by dsldsl**
When browsing the marketplace, I think its a nice value add to be able to scan the key stats for the legions to compare - both as a buyer I like to see it and as a seller I want to be able to showcase it.  (Even though it's in the stats box.)  Eventually this block of text can be compressed into a stats card, but for now I think the text is useful.

image

image

### CODE REVIEW

**Comment by dsldsl** [comment]
Oh, also - the filtering on the left side doesn't help address this use case because I'm not looking for something specific, just scanning between whats available for best value add.

**Comment by alecananian** [comment]
I feel like this is cluttering the UI a bit. Thoughts on moving that info into the (i) disclosure dialog?

**Comment by dsldsl** [comment]
I totally get that and the tradeoffs.  My use case is that I want to be able to 50 times per day scan the marketplace and look for buys.  That doesn't work for me in the popup dialog.  I have been using a version of this locally with my own marketplace front end for myself and friends to snipe prime legions.  Judging by the fact that the sales haven't been very intelligent I think there's a need for this data to help buyers.  The (selfish) reason that I'm sharing this now is that I'm considering selling some of my aux's soon as they are leveling up and I want to make sure I can justify a higher price tag for their traits.

so TL;DR - I understand the tradeoffs and if you don't want to add it, but I do think there's a value add to showing it on the collection page vs the (i) popup.  it's def more of a power user feature.

**Comment by jcheese1** [comment]
@dsldsl 

Yeah, I think this is good. I think those 3 are the most important.  More than this lets think of a UI.

Lets get this in

**Comment by jcheese1** [comment]
@allcontributors add @asharma13524 for code

**Comment by jcheese1** [comment]
KEK

**Comment by jcheese1** [comment]
@allcontributors add @dsldsl  for code

**Comment by dsldsl** [comment]
🎉 🎉 🎉
